tests/readPreference/readpreference-ctor_error-003.phpt

```diff
@@ -1,24 +1,24 @@
 --TEST--
-MongoDB\Driver\ReadPreference construction (invalid maxStalenessMS)
+MongoDB\Driver\ReadPreference construction (invalid maxStalenessSeconds)
 --FILE--
 <?php
 
 require_once __DIR__ . '/../utils/tools.php';
 
 echo throws(function() {
-    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, null, ['maxStalenessMS' => 1000]);
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, null, ['maxStalenessSeconds' => 1000]);
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
 
 echo throws(function() {
-    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, null, ['maxStalenessMS' => -1]);
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, null, ['maxStalenessSeconds' => -2]);
```

Comment:
Let's add test cases for `0` (another boundary case) and `42` (something less than 90) as well.